packages/libs/error-stack/src/lib.rs

```diff
@@ -399,25 +407,26 @@
 //!  Feature   | Description                                                    | implies | default
 //! -----------|----------------------------------------------------------------|---------|--------
 //!  `std`     | Enables support for [`Error`] and, on nightly, [`Backtrace`]   |         | enabled
-//!  `hooks`   |Enables the usage of [`set_display_hook`] and [`set_debug_hook`]| `std`   | disabled
 //! `spantrace`| Enables the capturing of [`SpanTrace`]s                        |         | disabled
 //!  `futures` | Provides a [`FutureExt`] adaptor                               |         | disabled
 //! `small`    | Enable optimizations for the memory footprint of [`Report`]    |         | enabled
 //!  `anyhow`  | Provides conversion from [`anyhow::Error`] to [`Report`]       |         | disabled
 //!   `eyre`   | Provides conversion from [`eyre::Report`] to [`Report`]        |         | disabled
+//!   `glyph`  | Provide color and use of box drawing characters in [`Debug`] output |    | enabled
 //!
 //! [`set_display_hook`]: Report::set_display_hook
 //! [`set_debug_hook`]: Report::set_debug_hook
 //!
 //! [`Error`]: std::error::Error
 //! [`Error::provide`]: std::error::Error::provide
 //! [`Backtrace`]: std::backtrace::Backtrace
-//! [`Display`]: std::fmt::Display
+//! [`Display`]: core::fmt::Display
+//! [`Debug`]: core::fmt::Debug
 //! [`SpanTrace`]: tracing_error::SpanTrace
 //! [`smallvec`]: https://docs.rs/smallvec
 #![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Cannot comment on that line. I think we should add the `unreachable_pub` warning to ensure the structs we are defining are available.

Reply:
We should probably warn on `unreachable_pub` and allow `https://rust-lang.github.io/rust-clippy/master/#redundant_pub_crate`.

Reply:
Yes, I think that's the best way forward! 👍🏻

Reply:
Deferring to PR, I changed it locally, thinking it would be a drive-by, but this will require some changes not only in `fmt` but in all other areas of the crate.